imports/api/lib/methods.js

```diff
@@ -39,18 +24,12 @@ Meteor.methods({
 
   getPosts() {
     this.unblock();
-    const url = 'https://blog.keenethics.com/?format=json';
-    const result = Meteor.http.call('GET', url, {
-      params: {
-        timeout: 3000,
-      },
-      headers: {
-        cookie: 'sid=1:Ae8BWDRjWeqWqHqGt3Ik6om0Z8MYiyFSgKLtnEqcwb9GzRN5xVtNgay6WT1Yx5PU;' +
-                'uid=f82ad8efa94',
-        'content-type': 'application/json',
-      },
+    const sorted = _.sortBy(fetchNews(), 'publishedDate').reverse().slice(0, 4);
+
+    _.each(sorted, (value, key) => {
+      sorted[key].publishedDate = moment(value.publishedDate).format('MMMM DD YYYY');
     });
-    const res = parseJSON(result.content);
-    return res;
+
+    return sorted;
```

Comment:
Please, let's make returned value an array - it seems we aren't using key for anything.